delta/app/src/main/resources/app.conf

```diff
@@ -199,6 +199,25 @@ app {
 
     # configuration for the persistence of progress of projections
     persist-progress-config = ${app.defaults.persist-progress-config}
+
+    # configuration for the automatic provisioning of projects.
+    automatic-provisioning {
+      enabled = false
+      permissions = ["resources/read", "resources/write", "projects/read"]
+      description = "Auto provisioned project"
+
+      # configuration of realms for automatic provisioning
+      # mapping between realm label and organization in which to provision the project
+      enabled-realms {
+        realm = "users-realm"
+      }
+      api-mappings {
+        # example API mapping
+        # documents = "https://bluebrain.github.io/nexus/vocabulary/defaultElasticSearchIndex"
+      }
+      vocab = ${app.http.base-uri}/
+      base = ${app.http.base-uri}/
```

Comment:
both `base` and `vocab` should be optional as we may want to fallback on the project defaults; please comment these and add a description.